app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<p style="color: green"><%= notice %></p>
+
+<%# <%= render @recipe %>
+<div class="recipe-show-page">
+  <div>
+    <div class="name_and_destroy">
+      <h2>
+        <%= link_to @recipe.name, @recipe %>
+      </h2>
+      <div class="prep-public-div">
+        <span>Preparation time: <%=@recipe.preparation_time%> min</span>
+        <% if @recipe.public == true %><span>Public ✔</span> <% else %> <span>Public ❌</span> <%end%>
+      </div>
+      <span>Cooking time: <%=@recipe.cooking_time%> min</span>
+      <span>Steps go here...</span>
+      <div class="sh-list_add-ingre-btn">
+        <%= button_to "Generate shopping list" %>
+        <% if @recipe.user_id == current_user.id %><%= link_to "Add ingredient", "/recipes/#{params[:id]}/recipe_foods/new" %><%end%>
```

Comment:
- [ ] Please make sure that adding an ingredient is working properly. Currently, it throws me this error
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/181864939-bff649eb-e0e5-464c-9f48-bf1ddfd7f57c.png)

Reply:
Hello @MahmoudBakr23 I do not know why it is throwing error in your side because it is working perfectly fine in my side. Just to be sure I just checked it again and it is working

Reply:
Resolved